openid-4-verifiable-presentations-1_0.md

```diff
@@ -1031,7 +1031,7 @@ The following is a non-normative example of the terms of use that may be defined
 }
 ```
 
-Federations that conform to those specified in [@OpenID.Federation] are identified by the `type` `urn:ietf:params:oauth:federation`. Individual federations are identified by the Entity Identifier of the trust anchor. If the federation decides to use trust marks as signs of whether an entity belongs to a federation or not then the federation is identified by the `type` `urn:ietf:params:oauth:federation_trust_mark` and individual federations are identified by the Entity Identifier of the trust mark issuer.
+Federations that conform to those specified in [@OpenID.Federation] are identified by the `type` `urn:openid:params:terms-of-use-type:federation`. Individual federations are identified by the Entity Identifier of the trust anchor. If the federation decides to use trust marks as signs of whether an entity belongs to a federation or not then the federation is identified by the `type` `urn:openid:params:terms-of-use-type:federation_trust_mark` and individual federations are identified by the Entity Identifier of the trust mark issuer.
```

Comment:
```suggestion
Federations that conform to those specified in [@OpenID.Federation] are identified by the `type` `urn:openid:params:federation`. Individual federations are identified by the Entity Identifier of the trust anchor. If the federation decides to use trust marks as signs of whether an entity belongs to a federation or not then the federation is identified by the `type` `urn:openid:params:federation_trust_mark` and individual federations are identified by the Entity Identifier of the trust mark issuer.
```
Adding "terms-of-use" to these URIs seems unnecessary and doesn't improve the clarity of meaning.  Once this change request is incorporated, I will approve the PR.

Reply:
These were being used only in some not especially clear or well specified application of VCDM's `termsOfUse` so I was trying to add some kind of meaning. But I'm not committed.